doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -233,3 +233,24 @@ The leader network data.
 * Packed-Encoding: `D`
 
 The stable leader network data.
+
+### PROP 5391: PROP_THREAD_JOINERS {#prop-thread-joiners}
+
+* Type: Read-Write
+* Packed-Encoding: `A(T(EDL))`
+
+Data per item is:
+
+* `E`: Extended/long address
+* `D`: PSKd
+* `L`: Timeout
+
+Passess Extended address and PSKd to the NCP in the commissioning process.
```

Comment:
How do we use this property when we don't have the extended address of the joiner (i.e. we only have the PSKD?)